.circleci/config.yml

```diff
@@ -18,17 +18,35 @@ jobs:
           name: Install dependencies
           command: |
               npm config set strict-ssl false
-              # Avoid downloading chromium unnecessarily
-              npm config set ignore-scripts true
-              npm install
-              npm config set ignore-scripts false
+              # puppeteer install script throws syntax errors for Node v4
+              # https://github.com/GoogleChrome/puppeteer/blob/f19e2ade0d4859435422131905e932abe3db199c/install.js#L66
+              if node --version | grep -q '^v4'; then
+                npm config set ignore-scripts true
+                npm install
+                npm config set ignore-scripts false
+              else
+                npm install
+              fi
+
+              # install latest chrome unstable version
+              if node --version | grep -q '^v6'; then
+                wget -q -O - https://dl-ssl.google.com/linux/linux_signing_key.pub | apt-key add -
+                echo "deb [arch=amd64] http://dl.google.com/linux/chrome/deb/ stable main" >> /etc/apt/sources.list.d/google.list
+                apt-get update
+                apt-get install -y --no-install-recommends google-chrome-unstable
+              fi
       - run:
           name: Pre-Test
           # ESLint only supports Node >=4
           command: |
-              NODE_VERSION=$(node --version)
-              if [[ ${NODE_VERSION:0:3} == "v6." ]]; then
+              if node --version | grep -q '^v6'; then
                 npm run lint;
+                # HACK: modify mochify to pass args to chrome to allow it to run as root
+                #   --no-sandbox --disable-setuid-sandbox
+                # See https://github.com/mantoni/mochify.js/issues/162
+                sed -i "s#'--allow-insecure-localhost'#'--allow-insecure-localhost', '--no-sandbox', '--disable-setuid-sandbox'#" node_modules/mochify/lib/chromium.js && echo "modified mochify/lib/chromium.js";
```

Comment:
This is gross but a necessary evil for now. This is the [result without the hack](https://circleci.com/gh/fearphage/Sinon.JS/73).

Reply:
We'll get this mixed in Mochify.